src/checker.rs

```diff
@@ -44,6 +44,7 @@ async fn do_request(client: &Client, url: &str) -> (RequestType, Result<Response
     (RequestType::Get, result)
 }
 
+/// Check if the reqwest error is a DNS error.
```

Comment:
```suggestion
/// Check if the request error is a DNS error.
```

Reply:
The name of the library is actually [`reqwest`](https://docs.rs/reqwest/latest/reqwest/) lol

Reply:
oh whoops, sorry!!

Reply:
feel free to revert/make a new pr i can re-approve

Reply:
nah it's fine, doesn't really matter.